cmd/rookflex/main.go

```diff
@@ -31,8 +32,8 @@ type result struct {
 func main() {
 	var r result
 	if err := cmd.RootCmd.Execute(); err != nil {
-		if e, ok := err.(*cmd.NotSupportedError); ok {
-			r.Status = e.Error()
+		if strings.HasPrefix(err.Error(), "unknown command") {
```

Comment:
would this error be coming from the cobra framework?  do they have a strong type we can check for, or a const string we can reference, instead of this hard coded string here?

Reply:
yes it comes from cobra framework. And no strong typed error objects. I looked :(